### PR TITLE
Change licence to AGPLv3 and update donations text

### PR DESCRIPTION
**Pull request by philipbelesky**
As discussed, this shifts Tabbycat to using a proper OSI licence. This is incompatible with our previous approach of requiring donations for profit/fundraising use, so I have updated the docs. I've also added a note about the possibility of dual-licensing for people who would like to avoid the gpl terms.

Happy to discuss here or over email; I just wanted to use the PR to ensure we are all onboard before merging.

### CODE REVIEW

File: docs/guide/comparisons.rst [hunk 1]
```diff
@@ -150,7 +150,7 @@ Other considerations
 
 **Stability and development**. Tournaman has been in use for over a decade and is generally considered to be stable and reliable. However, new features are relatively rarely added, and its being a native app means that it doesn't boast as many features as Tabbycat or Tabbie2.
 
-**Cost**. Tournaman is free to use. Tabbycat is free to use for not-for-profit, not-for-fundraising tournaments; tournaments for profit or fundraising must make a donation of A$1 per team. In addition, larger tournaments that run on Tabbycat's recommended web host (Heroku) may need to purchase an upgraded database service (the free tier has storage limits) which will cost around ~US$3 to use for the duration of a week-long tournament.
+**Cost**. Tournaman and Tabbycat are both free to use. Larger tournaments that run on Tabbycat's recommended web host (Heroku) may need to purchase an upgraded database service (the free tier has storage limits) which will cost around ~US$3 to use for the duration of a week-long tournament.
```
Comment (tienne-B) [inline docs/guide/comparisons.rst]: Maybe add a distinction between free and open.

Comment (tienne-B) [reply docs/guide/comparisons.rst]: Scratch that; I guess the next point goes into that.

**Comment by tienne-B** [comment]
I'm very open (enthusiastic even) to this change (and my commits may be re-licensed) for the GPLv3.

However, I am a bit skeptical about the relicensing for "proprietary" development. It goes against the "[copyleft](https://en.wikipedia.org/wiki/Copyleft)" aspect of GPLv3 (which is the GPLs "selling point"). Were there any hypothetical scenarios in which such a right is given?

**Comment by philipbelesky** [comment]
The dual license might end up being wholly hypothetical in practice, but the thinking was that it provides a means for companies/institutions to pay or be granted the right to be able to modify Tabbycat privately. From my understanding this is not a super unusual model, although there is some debate as to whether it is a 'true' open-source business model. 

In our case, if the proprietary changes were very beneficial to the broader project we could still reserve the right not to grant a proprietary licence. If the changes were not — say adding an importer that interfaces with a custom registration system, or adding a highly-tailored draw rule that would be of no use to others — it provides a means to allow those changes to happen without publicly disclosing the details.

Those scenarios are probably unlikely, but the thinking was to preserve the option for a limited form of proprietary license incase of future unforeseen circumstances where it is somehow desirable.

**Comment by tienne-B** [comment]
I don't really agree with these scenarios. The first case is to my mind the only reason why such rights would be asked for, and we are in agreement that it would not be allowed. The second case is for trivial modifications, for which I doubt people seeking "approval." Further, the GPL allows for the private modification and use of software (just not the non-open distribution). As such versions would not be distributed to its users (due to it being a web-application), asking for permission doesn't really apply. If they were to share their version for another tab team, they should be required to disclose source (but not necessarily upstream)

**Comment by philipbelesky** [comment]
Hmm, on that note, how would you / @czlee feel about the AGPL over the GPL? Given we are primarily a web application, rather than a library, the private modification + deployment scenario encapsulates the entire use case of the code base. The only case where distribution would be a pain is in the case of a major public fork, but the more likely scenario of a 'private' fork deployed as needed by a company/institution is also one where it would be valuable to have the source disclosed.

**Comment by czlee** [comment]
I moved the substantive discussion to an email thread.

One thing on documentation: To be consistent, I'd like us to maintain British spelling in all our documentation, please? The only exception would be LICENSE.md, because we obviously can't modify the GPL text.

**Review by tienne-B — COMMENTED** [review]
Might want to rebase this...

After discussion, we've moved towards using the AGPL with an opening for dual-licensing.

File: docs/about/licence.rst [hunk 2]
```diff
@@ -1 +1,17 @@
-.. include:: ../../LICENSE.rst
+=====================
+Licence and Donations
+=====================
+
+Licence
+=======
+
+Tabbycat is licenced under the terms of the `GNU Affero General Public License v3.0 <https://choosealicense.com/licenses/agpl-3.0/>`_. You may copy, distribute, and modify this software; however note that this licence requires (amongst other provisions) that any modifications you make to Tabbycat must be made public.
```
Comment (czlee) [inline docs/about/licence.rst:8]:
```suggestion
Tabbycat is licensed under the terms of the `GNU Affero General Public License v3.0 <https://choosealicense.com/licenses/agpl-3.0/>`_. You may copy, distribute, and modify this software; however note that this licence requires (amongst other provisions) that any modifications you make to Tabbycat be made public.
```

File: .github/README.md [hunk 3]
```diff
@@ -51,11 +51,13 @@ If you have any feedback or would like to request support, we'd love to hear fro
...
+Monetary donations are much appreciated and help us to continue the development and maintenance of Tabbycat. We suggest that tournaments donate at the level of $1 Australian dollar per team; especially if your tournament is run for profit or fund-raising purposes. More details [are available in our documentation](http://tabbycat.readthedocs.io/en/latest/about/licence.html).
 
 ## ©️ Licence
 
-We haven't released Tabbycat under an open-source licence, so there is no formal and general right to use this software. Nonetheless, you're welcome to freely use Tabbycat to help run a debating tournament. However, if your tournament is run as a for-profit or for-fundraising activity a donation to Tabbycat's maintainers is required. More details [are available in our licence information](http://tabbycat.readthedocs.io/en/latest/about/licence.html).
+Tabbycat is licenced under the terms of the [GNU Affero General Public License v3.0](https://choosealicense.com/licenses/agpl-3.0/). You may copy, distribute, and modify this software; however note that this license requires (amongst other provisions) that any modifications you make to Tabbycat must be made public.
```
Comment (czlee) [inline .github/README.md:58]:
```suggestion
Tabbycat is licensed under the terms of the [GNU Affero General Public License v3.0](https://choosealicense.com/licenses/agpl-3.0/). You may copy, distribute, and modify this software; however note that this licence requires (amongst other provisions) that any modifications you make to Tabbycat be made public.
```

"licensed" is a verb (or a past participle thereof) in this context, so it's spelt with an s

File: .github/README.md [hunk 4]
```diff
@@ -51,11 +51,13 @@ If you have any feedback or would like to request support, we'd love to hear fro
...
+
+If you wish to modify Tabbycat in a proprietary fashion we (the developers) are open to negotiating a dual-licence for this purpose. Please [contact us](http://tabbycat.readthedocs.io/en/latest/authors.html#authors) if this is the case.
```
Comment (czlee) [inline .github/README.md:60]:
```suggestion
If you wish to modify Tabbycat in a proprietary fashion we (the developers) are open to negotiating a dual licence for this purpose. Please [contact us](http://tabbycat.readthedocs.io/en/latest/authors.html#authors) if this is the case for you.
```

File: .github/README.md [hunk 5]
```diff
@@ -51,11 +51,13 @@ If you have any feedback or would like to request support, we'd love to hear fro
 
 Contributions are welcome, and are greatly appreciated! Details about how to contribute [are also outlined in our documentation](http://tabbycat.readthedocs.io/en/latest/about/contributing.html).
 
-We also invite new translations of the interface through [Crowdin](https://crowdin.com/project/tabbycat)! Get in touch for access to our translation platform.
+Monetary donations are much appreciated and help us to continue the development and maintenance of Tabbycat. We suggest that tournaments donate at the level of $1 Australian dollar per team; especially if your tournament is run for profit or fund-raising purposes. More details [are available in our documentation](http://tabbycat.readthedocs.io/en/latest/about/licence.html).
```
Comment (czlee) [inline .github/README.md:54]:
```suggestion
Monetary donations are much appreciated and help us to continue the development and maintenance of Tabbycat. We suggest that tournaments donate at the level of C$1 (1 Canadian dollar) per team; especially if your tournament is run for profit or fundraising purposes. More details [are available in our documentation](http://tabbycat.readthedocs.io/en/latest/about/licence.html).
```

**Review by czlee — COMMENTED** [review]
pedantic grammar stuff

File: tabbycat/templates/footer.html [hunk 6]
```diff
@@ -18,18 +18,20 @@ <h6 class="d-inline-block">
...
           {% blocktrans trimmed %}
-            Tabbycat is a source-available project developed by volunteers, and is free to use for tabbing non-profit, non-fundraising tournaments.
+            Tabbycat is an open-source project developed by volunteers and is free to use.
           {% endblocktrans %}
           {% if tournament and tournament.billable_teams > 0 %}
             {% blocktrans trimmed with amount=tournament.billable_teams %}
               For a tournament of this size, we suggest <a href="https://opencollective.com/tabbycat/contribute/licensing-29466/checkout?amount={{ amount }}">a donation of <strong>${{ amount }}</strong></a>.
             {% endblocktrans %}
             <a href="https://opencollective.com/tabbycat">
+            <a href="{% url 'donations' %}">
```
Comment (tienne-B) [inline tabbycat/templates/footer.html:28]:
```suggestion
```

File: tabbycat/templates/footer.html [hunk 7]
```diff
@@ -18,18 +18,20 @@ <h6 class="d-inline-block">
...
               {% trans "Learn more about donating." %}
             </a>
           {% else %}
-            {% blocktrans trimmed %}
-              <a href="https://opencollective.com/tabbycat">Donations to Tabbycat from individuals or organisations</a> are much appreciated.
+            {% url 'donations' as donations_url %}
+            {% blocktrans trimmed  %}
+              <a href="{{ donations_url }}">Donations to Tabbycat from individuals or organisations</a> are much appreciated and help us to continue to develop, support, and maintain Tabbycat.
```
Comment (tienne-B) [inline tabbycat/templates/footer.html:34]:
```suggestion
            {% blocktrans trimmed %}
              <a href="https://opencollective.com/tabbycat">Donations to Tabbycat from individuals or organisations</a> are much appreciated and help us to continue to develop, support, and maintain Tabbycat.
```

File: tabbycat/tournaments/templates/create_tournament.html [hunk 8]
```diff
@@ -13,11 +13,7 @@
 
   {% url 'donations' as donations_url %}
   {% blocktrans trimmed asvar donation %}
-    Tabbycat is free to use for non-profit and non-fundraising tournaments
-    (although donations are encouraged). If your tournament is run for profit
-    or for fundraising, please note that there is a
-    <a href="{{ donations_url }}">required payment</a>. For more details, see the <a href="http://tabbycat.readthedocs.io/en/stable/about/licence.html">
-    Tabbycat licence agreement</a>.
+    Tabbycat is free to use, although <a href="{{ donations_url }}">donations</a> are encouraged.
```
Comment (tienne-B) [inline tabbycat/tournaments/templates/create_tournament.html:16]:
```suggestion
  {% blocktrans trimmed asvar donation %}
    Tabbycat is free to use, although <a href="https://opencollective.com/tabbycat">donations</a> are encouraged.
```

**Review by tienne-B — COMMENTED** [review]
Any reason why we're re-adding the Donations page? Would just be simpler to keep redirecting to the OC page as we were doing.

**Review by tienne-B — APPROVED** [review]

**Comment by czlee** [comment]
I squashed it to one commit (slightly forcefully—I used a soft reset to avoid dealing with intermediate conflicts).

**Review by czlee — APPROVED** [review]

**Pull request merged**